ExercisesA/exercises.js

```diff
@@ -133,4 +133,10 @@ var arrayOfSciFiBooks = ['Dune: Frank Herbert',
 
 **/
 
-//YOUR CODE GOES HERE
+/** Also added this to the html file so it would work**/ 
```

Comment:
I don't think you need to add this to the HTML file or have the script tags. I think the script alone works, since we're loading exercises.js at the end of the HTML file.

I also didn't look into jQuery, so I don't really know what I'm talking about : - )